primaryHandler.go

```diff
@@ -66,6 +66,7 @@ type App struct {
 var (
 	errGettingPartnerIDs         = errors.New("unable to retrieve PartnerIDs")
 	errAuthIsNotOfTypeBasicOrJWT = errors.New("auth is not of type Basic of JWT")
+	basicstr                     = "basic"
```

Comment:
Normally, we don't want to specify the type (like "string") in the variable name.  Maybe we can call this `basicType` or `basicTokenType`?